### ISSUE TITLE
Curate 6w worth of events

### ISSUE THREAD

**Issue by pcuci** [bettermtl/bettermtl.github.io#2]
Show what the platform is capable of when used by a fantastic community

**Comment by pcuci** [comment]
I added new activities, need the copy updated.

Please use bettermtl.com as golden copy, I'll copy text from here to meetup/eventbrite/facebook @David-Ro

**Comment by pcuci** [comment]
Added the prototyping workshops (in English only), need translation: http://bettermtl.com

@David-Ro - which other tech communities should we bootstrap with?

Tech meetups, hackathons, other startup/networking events?

I need more real activities listed to start seeing patterns between all of them, maybe we can normalize some data so it's shared between events/translations, or other non-easily seen structural elements that only surface with more content.

**Comment by David-Ro** [comment]
I'm going to change the label name of places by pre-pending or pre- positioning a particle, so that the order of displaying places to newbie viewer fits our current model and needs

**Comment by pcuci** [comment]
@David-Ro describe what you want first, pre-pending while disregarding the logic of front-matter will only complicate things, better to [normalize](https://en.wikipedia.org/wiki/Database_normalization) data if this is core feature

**Comment by David-Ro** [comment]
```
                                                                                  I want the display / sort to follow a logic, other than alphabetical. So, geographic sort is a better beginning, to show this in April and May 2016. So, maybe we need a new item in addition to the one now (not shown to the public)  currently used to create the sort.  (Geofenced sites can be made separately, each with a separate name, separate identity, separate existence. Yes they are good experiments). I've seen you put two pieces of data into a single field and call it normalized. So, I can prepend in order to have the display order that I want to show to our current crowd who are all Plateau and downtown. Notman House must be first of the list or else we have to make a new site (name, identity, existence) for all our recurring dealings with the #hackthehouse crowd in the upcoming ioT integration season.  Makes sense ? 
```

**Comment by pcuci** [comment]
> put two pieces of data into a single field and call it normalized.

That's incorrect. Duplicating data on purpose is to **de-normalize** with good reasons (e.g.: data consistency, performance considerations).

Easier to add another front-matter data element, call it `sort_order` or `priority_no` or `vip_number`, and we can sort the locations accordingly. Keep concerns separate: order of display has nothing to do with address of location; therefore we split. We normalize the data.

**Comment by David-Ro** [comment]
```
                                                                                  Screen_display_order ok
```
